esmvaltool/utils/recipe_test_workflow/app/get_esmval/opt/rose-app-metoffice.conf

```diff
@@ -1,7 +1,11 @@
 [command]
-default=clone_latest_esmval.sh
+default=echo "Cloning ESMValCore and ESMValTool into ${ESMVAL_DIR}"
```

Comment:
I got a `[WARN] command not defined` message in the `job.err` without this section. The documentation states that [default is required](https://metomi.github.io/rose/doc/html/api/configuration/application.html#rose:conf:rose-app.conf[command]).

Reply:
Thanks @alistairsellar! 🎉 Change made in f74bc1644 😊